patterns/templates/jolla-hw-adaptation-@DEVICE@.yaml

```diff
@@ -75,5 +75,8 @@ Requires:
 #- qt5-qtmultimedia-plugin-mediaservice-irisradio
 #- jolla-mediaplayer-radio
 
+# NFC for devices using Android 8 or newer as base
+#- nfcd-binder-plugin
```

Comment:
Would be nice to add an example of the settings page: https://github.com/mer-hybris/droid-config-sony-nile/blob/2da0d49bca19d3cd42e13746c7724cdaf7089a1e/patterns/jolla-configuration-h4413.yaml#L20